lib/MediaConnection/RTCPeerMediaConnection.js

```diff
@@ -0,0 +1,171 @@
+const EventEmitter = require('events').EventEmitter;
+
+module.exports = class RTCPeerMediaConnection extends EventEmitter {
+
+    constructor(pcConfig, iceConnectionStateCallback)
+    {
+        super();
+        this.rtcPeerConnection = new RTCPeerConnection(pcConfig);
+        this.rtcPeerConnection.addEventListener('iceconnectionstatechange', () => {
+            iceConnectionStateCallback(this.rtcPeerConnection.iceConnectionState)
+        });
+    }
+
+
+    createSdpOffer(options) {
+
+        return this.rtcPeerConnection.signalingState === 'stable' ? this.rtcPeerConnection.createOffer(options)
+            .then(offer => {
+                return this.setLocalSdp(offer);
+            }) : Promise.resolve();
```

Comment:
This method is supposed to return a SDP, but here it's returning nothing.

Reply:
Yes, the lib don't use the return value. The same applies to 'createSdpAnswer'.
The lib get the data from the getLocalSdp function.

Reply:
Let's make `createSdpOffer()` return our local SDP always. So we don't unnecessarily mandate to always call `getLocalSdp()` after `createSdpOffer()`.